models/org_team.go

```diff
@@ -34,6 +35,64 @@ type Team struct {
 	Units       []*TeamUnit `xorm:"-"`
 }
 
+// SearchTeamOptions holds the search options
+type SearchTeamOptions struct {
+	UserID      int64
+	UserIsAdmin bool
```

Comment:
`UserIsAdmin` is not used? I guess the router takes care of the permissions, but why is it here?

Reply:
Removed